airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -60,9 +60,13 @@ properties:
     description: the Airbyte Protocol version supported by the connector
   normalizationRepository:
     type: string
+    description: an optional field indicating the name of the repository to be used for normalization. If the value of the flag is NULL - normalization is not used.
   normalizationTag:
     type: string
+    description: an optional field indicating the tag of the repository to be used for normalization.
   supportsDbt:
     type: boolean
-  integrationType:
+    description: an optional flag indicating whether DBT is used in the normalization. If the flag value is NULL - DBT is not used.
```

Comment:
Should we to default to false? The behavior would be the same and that could remove some null ptr exceptions.

Reply:
I tried leaving the default value here, but it overwrites the values that come from the definition file. I haven't figured out why this is happening yet, but it gives the wrong values to the actor_definition table when starting and reading information from the file. 

If you are afraid NPE - in all possible places, I added a check for NULL to prevent it from occurring.